app/src/components/items/uv/universalViewer.tsx

```diff
@@ -14,9 +14,28 @@ export type UniversalViewerProps = {
   onChangeManifest?: (manifest: string) => void;
 };
 
+const handleOnClick = (e) => {
+  console.log("e is: ", e);
+  if (e.target === "div.openseadragon-canvas") {
```

Comment:
Update this to `if (e.target.className === "openseadragon-canvas") {`

Reply:
This will bring it into the block and it adds the proper `position` and `zindex` values to the buttons, but it _still_ doesn't completely work.